content/kr/docs/tutorials/_index.md

```diff
@@ -7,65 +7,65 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-이번 섹션에는 Kubernetes 설명서 자습서가 포함되어 있습니다. 튜토리얼에서는 보다 보다 큰 방향을 갖고 접근합니다. 튜토리얼에는 여러 섹션이 있으며, 섹션별로 진행할 단계가 있습니다. 각 자습서를 살펴보기 전에 나중에 참조 할 수 있도록 표준화 용어집 페이지도 살펴보시기 바랍니다. 
+이번 섹션에는 Kubernetes 튜토리얼이 포함되어 있습니다. 튜토리얼에서는 개별 [작업](/docs/tasks/)에 국한되지 않고 보다 넓은 시야에서 접근합니다. 튜토리얼에는 여러 섹션이 있으며, 섹션별로 진행할 단계가 있습니다. 각 튜토리얼를 살펴보기 전에 나중에 참조할 수 있도록 [표준 용어집 페이지](/docs/reference/glossary/)도 살펴보시기 바랍니다. 
 
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## 기본
 
-* [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) 여러분이 Kubernetes 시스템을 이해하고 기본 Kubernetes 기능을 시도할 때 도움을 주는 대화식 튜토리얼입니다.
+* [Kubernetes 기초](/docs/tutorials/kubernetes-basics/) 여러분이 Kubernetes 시스템을 이해하고 기본 Kubernetes 기능을 시도할 때 도움을 주는 대화식 튜토리얼입니다.
 
-* [Scalable Microservices with Kubernetes (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
+* [Kubernetes를 사용한 확장성있는 마이크로 서비스 (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
 
-* [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
+* [Kubernetes 소개 (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
 
-* [Hello Minikube](/docs/tutorials/hello-minikube/)
+* [안녕 Minikube](/docs/tutorials/hello-minikube/)
```

Comment:
IMO just keeping "Hello Minikube" is fine since "Hello World" is a widely-known term for developers even in Korea - your thoughts?